### PR TITLE
feat: login signup with auth

### PR DESCRIPTION
**Pull request by bishnoimukesh**
# Change logs:

- feat: added toast
- feat- added auth
- feat- added login signup page layout

### CODE REVIEW

File: src/pages/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                >
+                    Login
+                </Button>
+                <Link as={ReachLink} to="/signup" fontSize={'16'}>
+                    Create an Account
+                </Link>
+                </Box>
+                <Button
+                type={"button"}
+                // rightIcon={<ArrowForwardIcon />}
```
Comment (aurobindoGupta) [inline src/pages/Login.jsx:91]: remove commented code

**Review by aurobindoGupta — APPROVED** [review]
- nice work, clean code.
Keep it upp

File: src/pages/Login.jsx (hunk 1 above)
Comment (bishnoimukesh) [reply src/pages/Login.jsx:91]: ok

**Review by ckahub — APPROVED** [review]
* The UI looks great.
* Clean folder structure.
* Don't see any breaking changes code looks mergeable.

**Pull request merged**